s3torchconnector/tst/e2e/models/lightning_transformer.py

```diff
@@ -0,0 +1,26 @@
+#  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#  // SPDX-License-Identifier: BSD
+from typing import Dict, Any
+
+import lightning as L
```

Comment:
Why not do a from import LightningModule here given that we're only using this once?